lib/TH/THTensor.c

```diff
@@ -7,6 +7,18 @@
 #include "THTensorDimApply.h"
 #include "THMath.h"
 
+inline double frac(double x) {
```

Comment:
I wonder if those functions shouldn't go to the newly created [THMath.h](https://github.com/torch/torch7/blob/master/lib/TH/THMath.h). What do you think ?